### ISSUE TITLE
"Components: Interfaces and Adapters" uses old-style "implements" API

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6269]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)| @hynek reported|
|-|-|
|Trac ID|trac#6269|
|Type|task|
|Created|2013-01-20 12:42:04Z|
|Branch|https://github.com/twisted/twisted/tree/component-doc-implementer-6269|

Since we’re moving Twisted off "implements()" and "implementsOnly()", the howtos should reflect that.

Attachments:

* [use-decorator-base-interface.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/8ed/8edc876b9222f23379f666107cf18b6d810e89c5/1817688efeed93a307a76e930850317bb812c44f.patch) (6365 bytes) - added by hynek on 2013-01-20 12:47:32Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__6269 6269
type__task task
reporter__hynek hynek
priority__normal normal
milestone__ 
branch__branches_component_doc_implementer_6269 branches/component-doc-implementer-6269
branch_author__tomprince tomprince
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation_easy documentation easy
time__1358685724000000 1358685724000000
changetime__1358698101000000 1358698101000000
version__None None
owner__tom_prince tom.prince
cc__hynek
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_2"></a>|@tomprince commented|
|-|-|

(In [36868]) Branching to component-doc-implementer-6269

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_3"></a>|@tomprince set owner to @tomprince|
|-|-|

This looks good. I'll merge, and file a ticket to change the rest of the documentation likewise.

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_4"></a>|@tomprince set status to `closed`|
|-|-|

(In [36870]) Merge branches/component-doc-implementer-6269: "Components: Interfaces and Adapters" howto now uses zope.interface's decorator-based API.

Author: hynek
Reviewers: tom.prince
Fixes: [#6269](https://github.com/twisted/twisted/issues/6269)

Since we’re moving Twisted off "implements()" and "implementsOnly()", the howtos should reflect that.
